src/components/App/App.js

```diff
@@ -1,13 +1,53 @@
 import React from 'react';
 import styles from './App.css';
+import Header from '../Header/Header';
+import Phrase from '../Phrase/Phrase';
+import Alphabet from '../Alphabet/Alphabet'
+
+class App extends React.Component {
+    state = {
+        id: '',
+        category: '',
+        encodedPhrase: '',
+        failsCounter: '',
+    }
+    fetchPhrase () {
+        const url = "http://localhost:3000/phrases/new"
```

Comment:
You probably don't need the 'localhost' part - try to use just '/phrases/new' or 'phrases/new'

Reply:
You are right, it is not needed.